src/main/java/seedu/address/logic/commands/SaveSortPreferenceCommand.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Saves the sort preference to UserPrefs
+ */
+public class SaveSortPreferenceCommand extends Command {
+
+    public static final String MESSAGE_SUCCESS = "Your preference has been saved. "
+            + "Relaunch, the application to view the changes";
+
+    public static final String COMMAND_WORD = "save_sort";
+
+    public static final String ALT_COMMAND_WORD = "svp";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Saves your desired sorting preference as priority "
+            + "or by last seen date based on the parameter.\n"
+            + "Parameters: high/low\n"
```

Comment:
Maybe high / low would be more readable in the display panel

Reply:
I agree, I was following the format of the other commands. Perhaps we can create and issue for this and resolve this issues for all the Command classes involved after v1.4?